scenes/game/local_game.gd

```diff
@@ -3854,8 +4014,13 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			# This effect is expected to be mid-strike.
 			assert(active_strike)
 			var card_name = card_db.get_card_name(card_id)
-			_append_log_full(Enums.LogType.LogType_Effect, performing_player, "boosts and sustains %s." % card_name)
 			performing_player.strike_stat_boosts.move_strike_to_boosts = true
+			if 'sustain' in effect:
```

Comment:
there were a bunch of effects using boost_this_then_sustain already that expect sustain to be the default.
Does this change that behavior? Did you go back and change those? I only scrolled through lond's changes in card defs.
I'd prefer to maybe just update all the cards and perhaps even change the name of this effect to something less leading?

Reply:
Oh, i see move_strike_to_boosts_sustain defaults to true.
Not sure how I feel about that...
I think at least put a comment here because otherwise we'll just never know what's going on when we come back later.

Reply:
maybe even make the extra thing put in the card definition "block_sustain": true? or "dont_sustain" or "dont_actually_sustain" haha